### PR TITLE
Parity Signer Latest App and Companion 

### LINKED ISSUE #2987: Add more info on Parity Signer with new version
Specifically let's include instructions for adding new parachains with latest version of Parity Signer.

From watercooler:
UPDATE: Parity Signer v5.0.0 is available in Apple Store! Remove the current version of Signer before installing the new one! Make sure you have your mnemonics!!

You can also use this extension specifically designed for Signer:
https://parity.link/signer-companion
And you can update the metadata from here when there's a runtime upgrade (v5 has the latest metadata):
https://metadata.parity.io/
For the moment it has metadata only for Polkadot, Kusama and Westend. You can add custom chains using this crate:
https://github.com/paritytech/parity-signer/tree/master/rust/generate_message
but honestly the process is beyond my understanding, so if someone could dumb it down that would be great. Or if Parity could start adding more chains in the site above.

### PR DESCRIPTION
**Pull request by DrW3RK**
Add info on Parity Signer Companion and the changes in 5.x.x app version

### CODE REVIEW

**Comment by DrW3RK** [comment]
I have added the preliminary information regarding the Parity Signer companion and the new Parity Signer app and how it differs from the older versions. @DannyS03 Let me know if you intend to work on the Parachains part. If not, I will take the ownership of that part of the linked issue as well. Thanks!

**Comment by DrW3RK** [comment]
Also, waiting for a response on this issue to refine the instructions further - https://github.com/paritytech/parity-signer-companion/issues/100

**Comment by salmad3** [comment]
> I have added the preliminary information regarding the Parity Signer companion and the new Parity Signer app and how it differs from the older versions. @DannyS03 Let me know if you intend to work on the Parachains part. If not, I will take the ownership of that part of the linked issue as well. Thanks!

Good with either or - I can work on the parachains part as part of the reviewal for the same page

**Comment by DrW3RK** [comment]
@DannyS03 Yeah, let's work on this together. There is no tutorial available for this. Just this readme file with some examples. Will have to test it out on a couple of parachains and then draft a tutorial for it. https://github.com/paritytech/parity-signer/tree/master/rust/generate_message

**Comment by nuke-web3** [comment]
I notice https://wiki.polkadot.network/docs/learn-account-restore#parity-signer - is this PR still a WIP?

**Comment by DrW3RK** [comment]
@NukeManDan It had a blocker https://github.com/paritytech/parity-signer-companion/issues/100 and was on the back burner. We will get to this soon.

**Comment by DrW3RK** [comment]
Videos and Support articles created here https://wiki.polkadot.network/docs/learn-account-generation#parity-signer

PR no longer relevant
